ccpp/data/GFS_typedefs.meta

```diff
@@ -716,6 +730,30 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+[dust12m_in]
+  standard_name = fengsha_dust12m_input
```

Comment:
What does `12m` mean? this looks like it is monthly data for 5 variables for every point, based on the dimensions. Would `monthly_fengsha_dust_input` make sense as a standard name?

Reply:
I think that's the name of the dataset, but I'll let @haiqinli respond.

Reply:
@grantfirl Yes, the monthly data for the 5 variables of dust emission. We combined the 12-month data into one single file.